scripts/ci_test_common.sh

```diff
@@ -15,9 +15,11 @@ make -j2 iot_tests_common
 # Run common tests.
 ./output/bin/iot_tests_common
 
-# Rebuild in static memory mode.
-cmake .. -DIOT_BUILD_TESTS=1 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_FLAGS="$CMAKE_FLAGS -DIOT_STATIC_MEMORY_ONLY=1"
-make -j2 iot_tests_common
+if [ "$RUN_TEST" != "coverage" ]; then 
+    # Rebuild in static memory mode.
```

Comment:
I don't understand the logic here.  Why does a coverage test exclude static memory mode?  The answer might prove useful to others, so perhaps we could put it in a readme somewhere.

Reply:
+1